### PR TITLE
[10.0][MIG] account_due_list_days_overdue

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Technical and functional review

**Comment by JordiBForgeFlow** [comment]
@darshan-serpent Can you please review?

File: account_due_list_days_overdue/models/account_overdue_term.py [hunk 1]
```diff
@@ -4,9 +4,9 @@
 # © 2016 Therp BV (<http://therp.nl>).
```
Comment (serpentcs-dev1) [inline account_due_list_days_overdue/models/account_overdue_term.py]: @mreficent Can you please modify the Copyrights as per new [standards](https://github.com/OCA/maintainer-tools/blob/master/template/module/models/model_name.py#L2)

`# Copyright 2016 Therp BV (<http://therp.nl>).`

**Review by serpentcs-dev1 — APPROVED** [review]
Code & Functional Review LGTM :+1:

**Pull request merged**